src/core/reducers/addons.js

```diff
@@ -0,0 +1,15 @@
+const initialState = {
+  wat: {slug: 'wat', title: 'Wat is this?'},
+  foo: {slug: 'foo', title: 'The foo add-on'},
+  food: {slug: 'food', title: 'Find food'},
+  bar: {slug: 'bar', title: 'The bar add-on'},
+};
+
+export default function addon(state = initialState, action) {
```

Comment:
Would be nice to tweak the style settings so we don have spaces around default args e.g. so we have `state=initialState` instead of `state = initialState`.